packages/SketchMorph2-Core.package/MorphicMonet.class/instance/handleNoModifierKeystroke..st

```diff
@@ -0,0 +1,11 @@
+event handling
+handleNoModifierKeystroke: aKeyboardEvent
+	aKeyboardEvent keyCharacter caseOf: {
+		[$+] -> [self clickMenuItemWith: M2ZoomIn label].
+		[$-] -> [self clickMenuItemWith: M2ZoomOut label].
+		[$p] -> [self clickMenuItemWith: M2Pencil label].
+		[$e] -> [self clickMenuItemWith: M2Eraser label].
+		[$t] -> [self clickMenuItemWith: M2TextTool label].
+		[$f] -> [self clickMenuItemWith: M2FillArea label]
+	} otherwise: [^ aKeyboardEvent "no hit"].
+	^ aKeyboardEvent "no hit"
```

Comment:
Don't think that `"no hit"` is needed twice here. Why do we have two return statements anyway, doesn't the latter one cover the prior one?